packages/web/components/chat/Chat.tsx

```
@@ -0,0 +1,76 @@
+'use client';
+
+import React, { memo, useCallback } from 'react';
+import { TimelineView } from '@/components/timeline/TimelineView';
+import { MemoizedChatInput } from '@/components/chat/MemoizedChatInput';
+import { useSessionEvents, useSessionAPI } from '@/components/providers/EventStreamProvider';
+import { useAgentContext } from '@/components/providers/AgentProvider';
+import type { ThreadId, AgentInfo, LaceEvent } from '@/types/core';
+
+interface ChatProps {}
+
+export const Chat = memo(function Chat({}: ChatProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)